webservices/args.py

```diff
@@ -126,7 +126,7 @@ def make_seek_args(field=fields.Int, description=None):
     }
 
 names = {
-    'q': fields.Str(required=True, description='Name (candidate or committee) to search for'),
+    'q': fields.List(fields.Str, required=True, description='Name (candidate or committee) to search for'),
```

Comment:
Just checking to make sure my assumption is correct on these changes - this is being changed from accepting a single value for a key to being able to support multiple values for the same key?